src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -466,16 +474,16 @@ export default class RightPanel extends React.PureComponent<Props, State> {
                           maxLength={ITEM_DESCRIPTION_MAX_LENGTH}
                           onChange={this.handleChangeDescription}
                         />
-                        {item.type === ItemType.WEARABLE && (
-                          <Select<WearableCategory>
-                            itemId={item.id}
-                            label={t('global.category')}
-                            value={(data as WearableData)!.category}
-                            options={this.asCategorySelect(item.type, wearableCategories as WearableCategory[])}
-                            disabled={!canEditItemMetadata}
-                            onChange={this.handleChangeCategory}
-                          />
-                        )}
+
+                        <Select<WearableCategory | EmoteCategory>
```

Comment:
This change is unrelated to this issue but allows updating the emote category in the item editor.